src/ontology/mondo-edit.obo

```diff
@@ -556327,7 +556328,7 @@ is_a: MONDO:0020573 {source="OMIM:617921", source="https://orcid.org/0000-0001-5
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0060670 {source="MONDO:CLINGEN"}
 relationship: excluded_subClassOf MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
 relationship: excluded_subClassOf MONDO:0005144 {source="OMIM:617921", source="https://orcid.org/0000-0001-5208-3432"} ! familial amyotrophic lateral sclerosis
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6323 {source="OMIM:617921"} ! KIF5A
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6323 ! KIF5A
```

Comment:
OMIM source removed correctly since this is susceptibility related https://omim.org/entry/617921 and https://omim.org/help/faq#1_6

Reply:
I don't think this gene association should have been removed. I will add it back.
Let's review how we handle susceptibilities.

Reply:
Ok, I added it to the "Next" Mondo Tech Working group [agenda](https://docs.google.com/document/d/1pKyAZsT1ZlZxxgkNBRucvQPwsxzDF0nhGC2VaiPLj_U/edit?pli=1&tab=t.0#heading=h.ea6yyy6f71w6)